nemo/collections/nlp/modules/common/megatron/module.py

```diff
@@ -38,10 +38,6 @@
 _BF16_TYPES = (torch.BFloat16Tensor, torch.cuda.BFloat16Tensor)
 
 
-def param_is_not_shared(param):
```

Comment:
There is still usage of this function in clip_grads.py, for example Line # 97. I think this function basically goes to use_fsdp path. Please keep it or do global search for all usages